manifests/m/Microsoft/PIX/2507.11/Microsoft.PIX.locale.en-US.yaml

```diff
@@ -0,0 +1,56 @@
+# Created with YamlCreate.ps1 v2.4.7 $debug=QUSU.CRLF.7-5-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: "2507.11"
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://www.microsoft.com/
+PublisherSupportUrl: https://devblogs.microsoft.com/pix/support-2/
+PrivacyUrl: https://privacy.microsoft.com/privacystatement
+Author: Microsoft Corporation
+PackageName: Microsoft PIX
+PackageUrl: https://devblogs.microsoft.com/pix/
+License: Proprietary
+# LicenseUrl:
+Copyright: Copyright (c) Microsoft Corporation. All rights reserved.
+CopyrightUrl: https://www.microsoft.com/legal/intellectualproperty/trademarks
+ShortDescription: Performance tuning and debugging for DirectX 12 games on Windows
+# Description:
+Moniker: pix
+Tags:
+- d3d12
+- dxr
+# Agreements:
```

Comment:
Is this section no longer necessary?

Reply:
The previous AgreementUrl is no longer available, and I couldn't find a new one. The installer doesn't have any agreement-related switches either.